### PR TITLE
chore: add ktlint in Travis

### LINKED ISSUE #639: Add ktlint in travis
**Describe the chore**
Add ktlint in travis

**Would you like to work on the issue?**

- [X] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by nikit19**
Fixes #639

### CODE REVIEW

**Comment by nikit19** [comment]
@iamareebjamal is this the correct way?
Travis fails currently because of wildcart imports and  Unexpected line break.
Should I fix them manually? or is it possible to ignore them in ktlint?

**Comment by nikit19** [comment]
@iamareebjamal Please review

**Comment by iamareebjamal** [comment]
Don't disable the wildcard imports check, we'll fix it in upcoming commits

**Comment by nikit19** [comment]
@iamareebjamal updated. Please revie

File: app/build.gradle [hunk 1]
```diff
@@ -119,9 +123,26 @@ dependencies {
...
     testImplementation 'org.threeten:threetenbp:1.3.7'
     androidTestImplementation 'com.android.support.test:runner:1.0.2'
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
 }
+
+task ktlint(type: JavaExec, group: "verification") {
+    description = "Check Kotlin code style."
+    classpath = configurations.ktlint
+    main = "com.github.shyiko.ktlint.Main"
+    args "src/**/*.kt"
```
Comment (iamareebjamal) [inline app/build.gradle]: Add `--android` before src

File: app/build.gradle [hunk 2]
```diff
@@ -119,9 +123,26 @@ dependencies {
...
+}
+
+task ktlintFormat(type: JavaExec, group: "formatting") {
+    description = "Fix Kotlin code style deviations."
+    main = "com.github.shyiko.ktlint.Main"
+    classpath = configurations.ktlint
+    args "-F", "src/**/*.kt"
```
Comment (iamareebjamal) [inline app/build.gradle]: Same here

**Comment by nikit19** [comment]
Updated

File: app/build.gradle [hunk 3]
```diff
@@ -137,12 +137,12 @@ task ktlint(type: JavaExec, group: "verification") {
     description = "Check Kotlin code style."
     classpath = configurations.ktlint
     main = "com.github.shyiko.ktlint.Main"
-    args "src/**/*.kt"
+    args "--androidsrc/**/*.kt"
```
Comment (iamareebjamal) [inline app/build.gradle]: There should be a space after android

Try running this locally

Comment (nikit19) [reply app/build.gradle]: Updated. The task keeps running, it never completes

**Comment by iamareebjamal** [comment]
Build is failing

**Comment by nikit19** [comment]
@iamareebjamal yes it will fail. As far as I know only way of making the build pass is adding those comments for wildcard imports. This is a bug in the library that has still not been solved
https://github.com/shyiko/ktlint/issues/48

Do you know any other alternative ?

**Comment by iamareebjamal** [comment]
OK, will see

**Comment by iamareebjamal** [comment]
Superseded by #769

**Pull request closed without merging**